src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/judgeapproval/ApproveDraftOrdersSubmittedHandlerTest.java

```diff
@@ -150,5 +163,4 @@ private static Stream<Arguments> invokeNotificationServiceForRefusalOrdersData()
                 )).build(), 0) // refusal order without id
         );
     }
-
 }
```

Comment:
I'd need to look at the code further to understand how the test is working with the method source.  Consider whether some javadoc would be useful to cover how the test is using the methodsource.